app/src/main/java/app/musikus/ui/theme/Theme.kt

```diff
@@ -74,43 +79,142 @@ fun MusikusTheme(
+private const val groupNameElem8: String = "Element 8"
+
+
+@Target(AnnotationTarget.FUNCTION, AnnotationTarget.CLASS)
+@Retention(AnnotationRetention.SOURCE)
+@Repeatable
+@Preview(name = "Light (redundant)", group = groupNameAllLight)
+@Preview(name = "Dark (redundant)", group = groupNameAllDark, uiMode = Configuration.UI_MODE_NIGHT_YES or Configuration.UI_MODE_TYPE_NORMAL)
+private annotation class MusikusPreviewCommonAnnotations
+
+@MusikusPreviewCommonAnnotations
+@Preview(name = "Light", group = "Whole Screen")
+@Preview(name = "Dark", group = "Whole Screen", uiMode = Configuration.UI_MODE_NIGHT_YES or Configuration.UI_MODE_TYPE_NORMAL)
+@Preview(name = "OldDevice", group = "Whole Screen",
+    device = "id:Nexus 5",
+    showSystemUi = true
+)
+@Preview(name = "Rotated", group = "Whole Screen",
+    device = "spec:parent=Nexus 6,orientation=landscape", showSystemUi = true)
+annotation class MusikusPreviewWholeScreen
+
+@MusikusPreviewCommonAnnotations
+@Preview(name = "Light", group = groupNameElem1)
+@Preview(name = "Dark", group = groupNameElem1, uiMode = Configuration.UI_MODE_NIGHT_YES or Configuration.UI_MODE_TYPE_NORMAL)
+annotation class MusikusPreviewElement1
+
+@MusikusPreviewCommonAnnotations
+@Preview(name = "Light", group = groupNameElem2)
+@Preview(name = "Dark", group = groupNameElem2, uiMode = Configuration.UI_MODE_NIGHT_YES or Configuration.UI_MODE_TYPE_NORMAL)
+annotation class MusikusPreviewElement2
+
+@MusikusPreviewCommonAnnotations
+@Preview(name = "Light", group = groupNameElem3)
+@Preview(name = "Dark", group = groupNameElem3, uiMode = Configuration.UI_MODE_NIGHT_YES or Configuration.UI_MODE_TYPE_NORMAL)
+annotation class MusikusPreviewElement3
+
+@MusikusPreviewCommonAnnotations
+@Preview(name = "Light", group = groupNameElem4)
+@Preview(name = "Dark", group = groupNameElem4, uiMode = Configuration.UI_MODE_NIGHT_YES or Configuration.UI_MODE_TYPE_NORMAL)
+annotation class MusikusPreviewElement4
+
+@MusikusPreviewCommonAnnotations
+@Preview(name = "Light", group = groupNameElem5)
+@Preview(name = "Dark", group = groupNameElem5, uiMode = Configuration.UI_MODE_NIGHT_YES or Configuration.UI_MODE_TYPE_NORMAL)
+annotation class MusikusPreviewElement5
+
+@MusikusPreviewCommonAnnotations
+@Preview(name = "Light", group = groupNameElem6)
+@Preview(name = "Dark", group = groupNameElem6, uiMode = Configuration.UI_MODE_NIGHT_YES or Configuration.UI_MODE_TYPE_NORMAL)
+annotation class MusikusPreviewElement6
+
+@MusikusPreviewCommonAnnotations
+@Preview(name = "Light", group = groupNameElem7)
+@Preview(name = "Dark", group = groupNameElem7, uiMode = Configuration.UI_MODE_NIGHT_YES or Configuration.UI_MODE_TYPE_NORMAL)
+annotation class MusikusPreviewElement7
+
+@MusikusPreviewCommonAnnotations
+@Preview(name = "Light", group = groupNameElem8)
+@Preview(name = "Dark", group = groupNameElem8, uiMode = Configuration.UI_MODE_NIGHT_YES or Configuration.UI_MODE_TYPE_NORMAL)
+annotation class MusikusPreviewElement8
+
+
```

Comment:
IMO theme properties which affect the actual app should not come after preview related code. If we can not move the preview code to separate files, we should at least keep it apart from the main code.

Reply:
I created a file `theme/PreviewStyling` which now contains all the Preview-related code